Search, badges, email fix

andrewhalle opened the pull request:
- simple search available for logged in user, searches by name of app, org, user
- pivotal tracker link, travis badge, cc badges
- fix emailer to send proper link out
- add repo url to engagements

armandofox commented on app/controllers/iterations_controller.rb:
```diff
@@ -53,7 +53,8 @@ def get_customer_feedback
         pf.save
 
-        url = "http://localhost:3000/feedback/#{eng.id}/#{iter.id}"
+        host = request.env["REQUEST_URI"].split("/")[2]
```
why not `request.host` or `request.host_with_port` ?
http://apidock.com/rails/ActionDispatch/Http/URL/host_with_port

armandofox commented on app/controllers/search_controller.rb:
```diff
@@ -0,0 +1,14 @@
+class SearchController < ApplicationController
```
can we standardize on 2 spaces, no use of tabs?

andrewhalle commented:
converted to 2 spaces

On Fri, Oct 27, 2017 at 1:45 PM, Armando Fox <notifications@github.com>
wrote:

> *@armandofox* commented on this pull request.
> ------------------------------
>
> In app/controllers/search_controller.rb
> <https://github.com/saasbook/esaas-engagements/pull/4#discussion_r147512931>
> :
>
> > @@ -0,0 +1,14 @@
> +class SearchController < ApplicationController
>
> can we standardize on 2 spaces, no use of tabs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/saasbook/esaas-engagements/pull/4#pullrequestreview-72625765>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AP8bXudoqQ4s5EinPrXmAO0k8wOhjCuwks5swkDkgaJpZM4QBZ7w>
> .
>

andrewhalle commented:
switched to request.host_with_port

On Fri, Oct 27, 2017 at 1:45 PM, Armando Fox <notifications@github.com>
wrote:

> *@armandofox* commented on this pull request.
> ------------------------------
>
> In app/controllers/iterations_controller.rb
> <https://github.com/saasbook/esaas-engagements/pull/4#discussion_r147512861>
> :
>
> > @@ -53,7 +53,8 @@ def get_customer_feedback
>          pf.iteration = iter
>          pf.save
>
> -        url = "http://localhost:3000/feedback/#{eng.id}/#{iter.id}"
> +        host = request.env["REQUEST_URI"].split("/")[2]
>
> why not request.host or request.host_with_port ?
> http://apidock.com/rails/ActionDispatch/Http/URL/host_with_port
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/saasbook/esaas-engagements/pull/4#pullrequestreview-72625673>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AP8bXljJ4BQ6Nx3QJRUhwJRe1xxRjRtwks5swkDNgaJpZM4QBZ7w>
> .
>
